### PR TITLE
Improve dgnss python bindings

### PR DESCRIPTION
**Pull request by ljbade**
Use `mk_sdiff_array` in the `dgnss_update` binding.

Also add some `DEBUG` logging for the dgnss functions. Useful for debugging problems with the filter. Based on @akleeman work

@swift-nav/firmware Ready for review.

### CODE REVIEW

File: src/dgnss_management.c [hunk 1]
```diff
@@ -251,11 +251,23 @@ void dgnss_update(u8 num_sats, sdiff_t *sdiffs, double receiver_ecef[3],
 {
   DEBUG_ENTRY();
   if (DEBUG) {
-    printf("sdiff[*].prn = {");
+    printf("dgnss_update\n");
```
Comment (mookerji) [inline src/dgnss_management.c]: Note that you can use log_debug?

Comment (ljbade) [inline src/dgnss_management.c]: Oh yeah that probably be a better choice!

**Comment by ljbade** [comment]
@mookerji I replaced the `printf` with `log_debug`

**Comment by mookerji** [comment]
Looks good, @ljbade ! Go ahead and merge whenever.

**Pull request merged**